### ISSUE TITLE
3 problem the control center, the symbol in the battery, the lock screen

### ISSUE THREAD

**Issue by quentinios** [andrewwiik/LittleX#38]
I do not have the elements on the Lockscreen as on the iPhone x with LittleX I am in 11.0.3 with an iPhone 7 Plus
![e027985e-eb1c-4fc2-901d-21b615963fb0](https://user-images.githubusercontent.com/36057708/35732066-609a4c9a-0818-11e8-9b28-59b81dc5b563.png)
 

I do not have the same control center as the iphone X with littleX I am in 11.0.3 with an iPhone 7 Plus
![34885a08-604e-4174-b6fc-a0da5f51720d](https://user-images.githubusercontent.com/36057708/35732159-b8802c04-0818-11e8-90ab-03a0a4a3a796.png)

I do not have the symbol in the battery when my iphone charge I am in 11.0.3 with an iPhone 7 Plus
![63167e5b-a432-4b6f-8b6f-bbbf10cc1479](https://user-images.githubusercontent.com/36057708/35732261-12a38780-0819-11e8-9a90-c87b0e0fe620.jpeg)

**Comment by andrewwiik** [comment]
These are not high priorities minus the control center as it does not affect the functionality of the tweak

**Comment by quentinios** [comment]
yes I understand, I'd like to have at least the symbol in the battery, the rest can wait
